.github/workflows/kuttl-e2e-test-1.20.yaml

```diff
@@ -13,15 +13,15 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: '1.16'
-      - name: Install kuttl
+      - name: Install kuttl and kubectl
         run: |
           sudo curl -Lo $KUTTL https://github.com/kudobuilder/kuttl/releases/download/v0.11.1/kubectl-kuttl_0.11.1_linux_x86_64
           sudo chmod +x $KUTTL
-          sudo curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          sudo curl -LO https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl
           sudo chmod +x kubectl
       - name: Build and prepare manifests
         run: make manifests
       - name: Build container
         run: make operator-image
       - name: Run kuttl 1.20
-        run: kubectl-kuttl test --kind-config tests/e2e/versions/kind-config-1.20.yaml --config tests/e2e/kuttl-test.yaml
+        run: sudo kubectl-kuttl test --config e2e/kuttl/operator-deployment-1.20.yaml
```

Comment:
How come we need `sudo` to run these tests?

Reply:
hmm.. not sure we do to be honest 🤔 let me try it without...